make_helpers/tbbr/tbbr_tools.mk

```diff
@@ -134,3 +135,6 @@ endif
 ifneq (${NS_BL2U},)
     $(eval $(call FWU_CERT_ADD_CMD_OPT,${NS_BL2U},--fwu,true))
 endif
+
+# Add the toc_flags to the FIP header (optional)
+$(if $(PLAT_TOC_FLAGS),$(eval $(call FIP_ADD_PAYLOAD,$(PLAT_TOC_FLAGS),--plat-toc-flags)))
```

Comment:
This looks unnecessary. You can just call FIP_ADD_PAYLOAD in your platform.mk instead of creating another build flag.